pyani_plus/db_orm.py

```diff
@@ -897,3 +898,27 @@ def db_comparison(  # noqa: PLR0913
     session.add(comp)
     session.commit()
     return comp
+
+
+def attempt_insert(
+    session: Session, values: list[dict], table: type[Comparison | Genome | Run]
```

Comment:
Would it be clearer to use ``type[Base]``, or simpler to restrict the function to just logging comparison table entries (currently the sole usage)?

Reply:
I don't know! 😊